lib/internal/util.js

```diff
@@ -10,7 +18,7 @@ exports.printDeprecationMessage = function(msg, warned) {
   if (process.throwDeprecation)
     throw new Error(msg);
   else if (process.traceDeprecation)
-    console.trace(msg);
+    console.trace(msg.replace(new RegExp(`^\\(${prefix}\\) `), ''));
```

Comment:
Not sure I understand why this double escaping is needed. Wouldn't a literal like `/\(${prefix}\) /` work?